### ISSUE TITLE
Will you open source training code and loss function code

### ISSUE THREAD

**Issue by MetaInsight7** [jiequancui/DKL#1]
Currently, only the trained model and evaluation system have been found in the open source files. Will the training code be made public？

**Comment by jiequancui** [comment]
Hi,

Thanks for your interest in our work.

Our paper is still in submission.

The code will be released soon.


Best,
Jiequan
________________________________
From: Meta ***@***.***>
Sent: Sunday, December 3, 2023 10:04
To: jiequancui/DKL ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [jiequancui/DKL] Will you open source training code and loss function code (Issue #1)


Currently, only the trained model and evaluation system have been found in the open source files. Will the training code be made public？

**Comment by MetaInsight7** [comment]
Thank you for your prompt response. Best of luck with the paper submission!

**Comment by jiequancui** [comment]
Hi,

Thanks for your interest in our work.
Our code is available.

Best,
